src/puppetlabs/puppetdb/anonymizer.clj

```diff
@@ -31,11 +32,10 @@
    (contains? event "old_value")
    (contains? event "resource_type")))
 
-(defn report?
+(pls/defn-validated report? :- report/report-schema
```

Comment:
actually this function doesn't do what it says anymore, it's just a wrapper around keywordize-keys